pkgs/data/fonts/apple-color-emoji/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "apple-color-emoji";
+
+  src = fetchurl {
+    url = https://github.com/samuelngs/apple-emoji-linux/releases/download/latest/AppleColorEmoji.ttf;
```

Please quote URLs. See https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md
```suggestion
    url = "https://github.com/samuelngs/apple-emoji-linux/releases/download/latest/AppleColorEmoji.ttf";
```